src/view/sidepanel/sidepanel.cpp

```diff
@@ -12,6 +12,8 @@ View::SidePanel::SidePanel::SidePanel(spt::Spotify &spotify, const lib::settings
 	title = new View::SidePanel::Title(this);
 	setTitleBarWidget(title);
 
+	titleToTabWidgetLookup = new QMap<std::string, QWidget*>();
```

Comment:
Using `new` here is unnecessary and causes a memory leak. Allocating objects on the heap is only required for widgets, which uses parents to avoid memory leaks.
Also, `QHash` is often preferred when order doesn't matter.